tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2022 Toitware ApS.
+# Use of this source code is governed by a Zero-Clause BSD license that can
+# be found in the tests/LICENSE file.
+
+file(GLOB TESTS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "*_test.toit")
+
+set(TOIT_EXEC "toit.run" CACHE FILEPATH "The executable used to run the tests")
+set(TPKG_EXEC "toit.pkg" CACHE FILEPATH "The executable used to install the packages")
+set(TEST_TIMEOUT 40 CACHE STRING "The maximal amount of time each test is allowed to run")
+
+add_custom_target(
+  "install-pkgs"
+  COMMAND "${TPKG_EXEC}" install
+  WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+)
+
+include(ProcessorCount)
+ProcessorCount(NUM_CPU)
+
+add_custom_target(
```

Comment:
You should expect this to be broken on Windows. See https://github.com/toitlang/toit/pull/793.

Reply:
Will do a follow-up PR.